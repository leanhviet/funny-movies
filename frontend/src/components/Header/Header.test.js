// Libs
import React from 'react'
import { shallow } from 'enzyme'

// Components
import { HeaderComponent } from './index'

describe('Header component', () => {
  const wrapper = shallow(<HeaderComponent />)

  it('should be defined Alert component', () => {
    expect(wrapper).toBeDefined()
  })

  // Testing Snapshot
  it('should be render snapshot Alert component', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
