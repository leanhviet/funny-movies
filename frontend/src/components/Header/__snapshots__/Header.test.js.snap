// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should be render snapshot Alert component 1`] = `
<div
  className="header-cmp"
>
  <Navbar
    bg="light"
    className="header-cmp__wrap"
    collapseOnSelect={false}
    expand="lg"
    variant="light"
  >
    <Link
      className="header-cmp__wrap__logo"
      to="/"
    >
      <h1>
        <i
          aria-hidden="true"
          className="fa fa-home"
        />
         Funny movies
      </h1>
    </Link>
    <div
      className="ml-auto header-cmp__wrap__btn-grp"
    >
      <Link
        to="/login"
      >
        <Button
          active={false}
          disabled={false}
          variant="outline-primary"
        >
          Login / Register
        </Button>
      </Link>
    </div>
  </Navbar>
  <hr
    className="header-cmp__line"
  />
</div>
`;
